[INFO   ] Logger: Record log in /storage/emulated/0/python3/NeroProgs/kivyMD/coresMD/.kivy/logs/kivy_21-05-30_3.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.8.3 (default, Jun 15 2020, 02:19:10) 
[GCC 9.3.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/bin/python3"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil, img_gif (img_ffpyplayer ignored)
[INFO   ] KivyMD: v0.104.1
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.0 V@415.0 (GIT@d39f783, I79de86aa2c, 1591296226) (Date:06/04/20)'>
[INFO   ] GL: OpenGL vendor <b'Qualcomm'>
[INFO   ] GL: OpenGL renderer <b'Adreno (TM) 308'>
[INFO   ] GL: OpenGL parsed version: 3, 0
[INFO   ] GL: Texture max size <8192>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Text: Provider: sdl2
[INFO   ] GL: NPOT texture support is available
[WARNING] Base: Unknown <android> provider
[INFO   ] Base: Start application main loop
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/python3/NeroProgs/kivyMD/coresMD/main.py", line 91, in <module>
[WARNING] stderr:     Palette().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/app.py", line 855, in run
[WARNING] stderr:     runTouchApp()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/base.py", line 504, in runTouchApp
[WARNING] stderr:     EventLoop.window.mainloop()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/window/window_sdl2.py", line 747, in mainloop
[WARNING] stderr:     self._mainloop()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/window/window_sdl2.py", line 479, in _mainloop
[WARNING] stderr:     EventLoop.idle()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/base.py", line 348, in idle
[WARNING] stderr:     Clock.tick_draw()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/clock.py", line 598, in tick_draw
[WARNING] stderr:     self._process_events_before_frame()
[WARNING] stderr:   File "kivy/_clock.pyx", line 427, in kivy._clock.CyClockBase._process_events_before_frame
[WARNING] stderr:   File "kivy/_clock.pyx", line 467, in kivy._clock.CyClockBase._process_events_before_frame
[WARNING] stderr:   File "kivy/_clock.pyx", line 465, in kivy._clock.CyClockBase._process_events_before_frame
[WARNING] stderr:   File "kivy/_clock.pyx", line 154, in kivy._clock.ClockEvent.tick
[WARNING] stderr:   File "kivy/_clock.pyx", line 86, in kivy._clock.ClockEvent.get_callback
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/weakmethod.py", line 56, in is_dead
[WARNING] stderr:     return self.proxy is not None and not bool(dir(self.proxy))
[WARNING] stderr: ReferenceError: weakly-referenced object no longer exists
